labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala

```diff
@@ -79,7 +79,7 @@ object HelloWorld {
 object HelloWorldClassAndObject {
 	def apply(initialText:String):HelloWorldClassAndObject = {
 		new HelloWorldClassAndObject {
-			val text="FixMe"
+			val text="initialText"
```

Comment:
I think you'd want to remove the quotes here, otherwise it'll make initialText a string literal instead of using the value passed in.